otx/algorithms/classification/task.py

```diff
@@ -127,16 +129,22 @@ def __init__(self, task_environment: TaskEnvironment, output_path: Optional[str]
         if self._task_environment.model is not None:
             self._load_model()
 
+    def _is_multi_label(self, label_groups: List[LabelGroup], all_labels: List[LabelEntity]):
+        """Check whether the current training mode is multi-label or not."""
+        # NOTE: In the current Geti, multi-label should have `___` symbol for all group names.
+        find_multilabel_symbol = ["___" in getattr(i, "name", "") for i in label_groups]
```

Comment:
It's just a question. Does it consider both Geti case and OTX case? Will `find_multilabel_symbol` get a list whose all elements are True in OTX alone case?

Reply:
Nice point. In the current OTX, we only consider the exported Geti dataset. So, Geti = OTX for multi-label and h-label cases.